client/signup/utils.js

```diff
@@ -195,12 +195,20 @@ export function getFirstInvalidStep( flowName, progress ) {
 	return find( getFilteredSteps( flowName, progress ), { status: 'invalid' } );
 }
 
-export function getCompletedSteps( flowName, progress ) {
+export function getCompletedSteps( flowName, progress, options = {} ) {
+	if ( options.shouldMatchFlowName ) {
```

Comment:
You and I will be familiar with what `shouldMatchFlowName` is doing (well, I'll probably forget in a week). Still, it might be worth adding a comment here as to why we're checking for it for the benefit of future archeologists. :)

Reply:
Great idea! I'll add it now.